app/assets/javascripts/application.js

```diff
@@ -255,7 +255,7 @@ function template_info(div, url) {
   // Use a post to avoid request URI too large issues with big forms
   $.ajax({
     type: "POST",
-    url: url + "&provisioning=" + build,
+    url: url + ((url.indexOf('?') == -1) ? '?' : '&') + "provisioning=" + build,
```

Comment:
my point is that since this bug is in contruction the url, we can be a bit more specific. for example:
```js
url="http://localhost:3000/hosts/template_used?x=1?&=2"
url.indexOf('?')
41
url="http://localhost:3000/hosts/template_used&x=1?y=2"
url.indexOf('?')
45
```
I realize that my solution is not useful either in this case:
when testing with URL 
```js
"http://localhost:3000/hosts/template_used&x=1?y=2"
(new URL(url)).search
"?y=2"
```
I guess this not worth the time, merging :)